app/controllers/distributions_controller.rb

```diff
@@ -27,6 +27,9 @@ def reclaim
   end
 
   def index
+    @highlight_id = session[:created_distribution_id]
+    session[:created_distribution_id] = nil
```

Comment:
You can combine this into a single line with the `Hash.delete` method :) 

https://ruby-doc.org/core-1.9.3/Hash.html#method-i-delete